### PR TITLE
Add category to OCL && do some small fix.

### PR DESCRIPTION
**Pull request by iskey**
related to #167

### CODE REVIEW

File: modules/database/src/main/java/org/eclipse/xpanse/modules/database/service/DeployResourceEntity.java [hunk 1]
```diff
@@ -30,14 +30,26 @@ public class DeployResourceEntity {
     @Id
```
Comment (swaroopar) [inline modules/database/src/main/java/org/eclipse/xpanse/modules/database/service/DeployResourceEntity.java:30]: can we remove this JPA annotation here?

**Review by swaroopar — CHANGES_REQUESTED** [review]
one minor comment added.

File: modules/database/src/main/java/org/eclipse/xpanse/modules/database/service/DeployResourceEntity.java (hunk 1 above)
Comment (iskey) [reply modules/database/src/main/java/org/eclipse/xpanse/modules/database/service/DeployResourceEntity.java]: yes, we can remove this.
I shouldn't follow the IDE's suggestion with "xxx should have primary key ".

**Review by swaroopar — APPROVED** [review]
/LGTM

**Comment by baixinsui** [comment]
look good.

**Pull request merged**